boot/zephyr/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  description: mcuboot test build sample
+  name: mcuboot
+
+tests:
+  sample.bootloader.mcuboot:
+    tags: bootloader_mcuboot
+    platform_whitelist:  nrf52840_pca10056
```

Comment:
Should we build it for another platform too? I know mcuboot supports many other Zephyr boards. Perhaps start with this one and then add more later?

Reply:
That my thought as well.

Reply:
The frdm_k64f should always work, so maybe that one as well?

Reply:
I agree, `nrf52840_pca10056` and `frdm_k64f` are the two boards I use for all my Zephyr testing.

Reply:
Added frdm_k64f. Are you using it to test serial recovery?

Reply:
^^ if yes - please give me proj.conf for it.

Reply:
I don't think I've used `boot_serial` with the `frdm_k64f`, but I'm gonna test it later and can send you the `proj.conf`; but it should not block this PR.

Reply:
It doesn't work, so not enabling it right now. Have disabled it now.